docs/advanced-topics/custom-transports.md

```diff
@@ -3,21 +3,21 @@ id: custom-transports
 title: Custom Transports
 ---
 
-The MLAPI uses Unity’s [UNet Transport Layer API ](https://docs.unity3d.com/Manual/UNetUsingTransport.html) by default and has support for custom Transports. 
+Netcode for Gameobjects (Netcode) uses Unity’s [UNet Transport Layer API ](https://docs.unity3d.com/Manual/UNetUsingTransport.html) by default and has support for custom Transports. 
```

Comment:
just a note on that, we might come back to this after UTP becomes the default transport for the SDK.